src/App.tsx

```diff
@@ -1,27 +1,86 @@
+import { useState } from 'react';
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoInfo } from './components/TodoInfo/TodoInfo';
+import React from 'react';
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState('');
```

Comment:
The `userId` state is initialized as an empty string. Since `userId` is later used as a number (e.g., `Number(userId)`), it might be more appropriate to initialize it as `null` or `undefined` to reflect its intended type more accurately.